js/central.js.php

```diff
@@ -52,22 +52,21 @@
 
             // get central list for plugin and insert in group tab
             $(".masonry_grid").each(function(){
-               var masonry_id = $(this).attr('id');
-
-               if (this.innerHTML.indexOf('escalade_block') < 0) {
-                  //prepare a span element to load new elements
-                  $(this).prepend("<div class='grid-item col-xl-6 col-xxl-4'><div class='card' id='escalade_block'></div></div>");
-
-                  //ajax request
-                  $("#escalade_block").load(CFG_GLPI.root_doc+"/"+GLPI_PLUGINS_PATH.escalade+'/ajax/central.php', function() {
-                     if ($("#escalade_block").html() == "") {
-                        $("#escalade_block").closest('.grid-item').remove();
-                     } else {
-                        var msnry = new Masonry('#'+masonry_id);
-                        msnry.layout();
-                     }
+               var masonry_id = $(".masonry_grid").attr('id');
+               var container = $(".masonry_grid");
```

Comment:
This code does not seems logical.

There is a `$(".masonry_grid").each()` loop, that suggets ther can be multiple items with `masonry_grid` class, but inside the loop, code suggest there is only one element with this class 'e.g. ``var container = $(".masonry_grid");``).

Reply:
fixed